### DISCUSSION TITLE
Nested Labels

### DISCUSSION THREAD

**Discussion by dtp555-1212** [doccano/doccano#1190] [Ideas]
The current version appears not to be able to handle nested labeling. (e.g. no way to mark a phrase and it parts). Is that a technical limitation or an artificial one? (e.g. would it be ‘possible’ to enhance the annotation tool to allow that?)

Thanks

**Comment by kris927b** [comment]
If you haven't found it already, I think what you are looking for is here: https://github.com/doccano/doccano/issues/544#issuecomment-640017994

So if you are up for altering some code the changes are there until an official update with the feature in mind. It seems to be around v1.5.0 (https://github.com/doccano/doccano/projects/11) 😄 

I have used it myself, and it works like a charm (only one level though). 😉 

Hope it helps.
